src/components/Meta.tsx

```diff
@@ -37,29 +45,23 @@ export default function render(
           "docs sectionType-" + props.sectionType + " " + props.category
         }
       />
+      {/* SEO content */}
+      <meta name="description" content={props.fields?.seoDescription}/>
+        
+      <meta property="og:image" content={ogImageURL} />
+      <meta name="twitter:image" content="image url from Contentful response" />
+      
+      <meta name="robots" content={props.fields?.robotsTag} />
+      <link rel="canonical" href={props.fields?.canonicalUrl}/>
+      
+      <script type="application/ld+json">
+      {/* // JSON from Contentful */}
```

Comment:
Add some more comments here about what this is for and how it connects to contentful in the future